content/en/continuous_integration/intelligent_test_runner/_index.md

```diff
@@ -23,8 +23,21 @@ Intelligent Test Runner is Datadog's test impact analysis solution. It allows yo
 
 By only running tests on relevant code, when tests fail, it is likely a legitimate failure that pertains to the modified code.
 
+## Limitations during beta
 
-## Setup
+During the beta of Intelligent Test Runner there are certain limitations:
+
+- Some of the environment variables required in the following sections are only required during the beta.
+- Intelligent Test Runner only works without the Datadog Agent.
```

Comment:
Can you add some more context to this bullet point? Are there steps they need to take if they were using Datadog Agent and want to implement ITR that they'd need to remove or disable it?
```suggestion
- Intelligent Test Runner only works without the Datadog Agent.
```

Reply:
I have added some more text